src/zenml/zen_server/deploy/helm/templates/custom-ca-configmaps.yaml

```diff
@@ -0,0 +1,20 @@
+{{- range $ca := .Values.zenml.certificates.customCAs }}
+{{- range $cm := list (dict "suffix" "") (dict "suffix" "-db-migration" "isDbMigration" true) }}
+---
+apiVersion: v1
+kind: ConfigMap
```

Comment:
`Secret` is more suitable for storing certs

Reply:
Duly noted, will switch to secret.